test/System/Vector2.test.cpp

```diff
@@ -271,14 +271,14 @@ TEMPLATE_TEST_CASE("[System] sf::Vector2", "", int, float)
         constexpr sf::Vector2f v(2.4f, 3.0f);
 
         CHECK(v.angle() == Approx(51.3402_deg));
-        CHECK(sf::Vector2f::UnitX.angleTo(v) == Approx(51.3402_deg));
-        CHECK(sf::Vector2f::UnitY.angleTo(v) == Approx(-38.6598_deg));
+        CHECK(sf::Vector2f{1.f, 0.f}.angleTo(v) == Approx(51.3402_deg));
```

Comment:
```suggestion
        CHECK(sf::Vector2f1.f, 0.f).angleTo(v) == Approx(51.3402_deg));
```
SFML's style convention is to use parens for constructor calls for non-aggregate types.

Reply:
Done.